modules/orca/main.tf

```diff
@@ -1,8 +1,3 @@
-locals {
-  # Used for Load Balancer, EC2 Service, and Container ports. Specifies how GQL will be hosted.
-  deploy_graphql = false
```

Comment:
Now that I'm documenting things, I needed an output variable to tie to. Seemed like the right time to remove the flag.

Reply:
so this means we will be deploying graphql from now on by default